frontend/src/components/pod/__snapshots__/PodDetails.stories.storyshot

```diff
@@ -58,44 +58,10 @@ exports[`Storyshots Pod/PodDetailsView Error 1`] = `
               >
                 <div
                   class="MuiGrid-root MuiGrid-item"
-                >
```

Comment:
These stories should now be tested with the AuthVisible true/false states. Otherwise we can't see these in the storybook anymore.

Reply:
Right. But the states being tested are the same as before, i.e. we need to make sure the UI looks the way it should for the status it has: no permissions. And this is the same as before (it's just the UI changes).

So we should add more stories for testing the UI when the state is such that the buttons must be visible. I think that can be added outside of this PR.